components/typography/typography.css

```diff
@@ -0,0 +1,92 @@
+.heading-1{
```

Comment:
typography code looks neat and good to merge

Reply:
Then please approve this too.